tests/DecisionServiceTests/DecisionServiceTest.php

```diff
@@ -77,7 +88,7 @@ public function testGetVariationReturnsNullWhenExperimentIsNotRunning()
 
         $pausedExperiment = $this->config->getExperimentFromKey('paused_experiment');
 
-        $this->decisionService = new DecisionService($this->loggerMock, $this->config);
+        
```

Comment:
why did u delete  $this->decisionService = new DecisionService($this->loggerMock, $this->config);
This supposed to be a part of this test case.

Reply:
got it. no comments. its already down.

Reply:
👍